terraform/volume.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_ebs_volume" "wp_content" {
+  # TODO deletion protection
```

Comment:
@afeld Just a nit... When a "type" property isn't specified in this resource, it's created as a "standard" volume. Default for creating a volume in the console is "GP2" (General Purpose SSD). I'd recommend adding:
type = "gp2"
...just to keep things standardized I guess ;)
Ref: [aws_ebs_volume](https://www.terraform.io/docs/providers/aws/r/ebs_volume.html)

Reply:
Along with that, you'll want encryption at rest enabled as well.